### PR TITLE
cleaned the code

### PR DESCRIPTION
**Pull request by aneshodza**

### CODE REVIEW

**Comment by aneshodza** [comment]
When resolving the conflict, add two } in the incoming change and then accept both changes. For some reason it doesnt save it when i resolve the conflict

**Comment by aneshodza** [comment]
upper comment still applies

File: src/components/CustomerStatements.vue [hunk 1]
```diff
@@ -287,13 +287,18 @@ export default {
 <style scoped lang="scss">
 @use "@/stylesheets/global.scss";
 
+@media screen and (max-width: 440px) {
+  .customer-container-sm {
+    margin: 0px 20px !important;
```
Comment (pandermatt) [inline src/components/CustomerStatements.vue:292]: why did u use important?

File: src/components/IMSButton.vue [hunk 2]
```diff
@@ -27,6 +27,12 @@ export default {
 <style scoped lang="scss">
 @use "@/stylesheets/variables.scss";
 
+@media screen and (max-width: 510px) {
+  a {
+    margin: 0px 10px !important;
```
Comment (pandermatt) [inline src/components/IMSButton.vue]: same here?

**Review by pandermatt — APPROVED** [review]
nice

**Pull request merged**